### ISSUE TITLE
centos - jps can't be executed.

### ISSUE THREAD

**Issue by gitava** [gitava/Big-Data-Study#16]
```
[vagrant@hdp-node-01 bin]$ jps
Error: Could not find or load main class sun.tools.jps.Jps
```

**Comment by gitava** [comment]
```
[vagrant@hdp-node-01 bin]$ sudo yum install java-1.8.0-openjdk-devel.x86_64
Failed to set locale, defaulting to C
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirrors.huaweicloud.com
 * extras: mirrors.huaweicloud.com
 * updates: mirrors.huaweicloud.com
Resolving Dependencies
--> Running transaction check
---> Package java-1.8.0-openjdk-devel.x86_64 1:1.8.0.252.b09-2.el7_8 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

========================================================================================================================
 Package                              Arch               Version                              Repository           Size
========================================================================================================================
Installing:
 java-1.8.0-openjdk-devel             x86_64             1:1.8.0.252.b09-2.el7_8              updates             9.8 M

Transaction Summary
========================================================================================================================
Install  1 Package

Total download size: 9.8 M
Installed size: 40 M
Is this ok [y/d/N]: y
Downloading packages:
java-1.8.0-openjdk-devel-1.8.0.252.b09-2.el7_8.x86_64.rpm                                        | 9.8 MB  00:00:01
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : 1:java-1.8.0-openjdk-devel-1.8.0.252.b09-2.el7_8.x86_64                                              1/1
  Verifying  : 1:java-1.8.0-openjdk-devel-1.8.0.252.b09-2.el7_8.x86_64                                              1/1

Installed:
  java-1.8.0-openjdk-devel.x86_64 1:1.8.0.252.b09-2.el7_8

Complete!
```

**Comment by gitava** [comment]
```
[root@hdp-node-01 bin]# find / -name jps
/etc/alternatives/jps
/usr/bin/jps
/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.252.b09-2.el7_8.x86_64/bin/jps
/home/vagrant/jdk/bin/jps
```

**Comment by gitava** [comment]
Solved under openjdk 1.8 not java jdk 1.7 under /home/vagrant/jdk.
```
# .bash_profile

# Get the aliases and functions
if [ -f ~/.bashrc ]; then
	. ~/.bashrc
fi

# User specific environment and startup programs
# add pig bin folder to path for pig installation
# add the following one to run jps under openjdk
#/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.252.b09-2.el7_8.x86_64/bin/
PATH=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.252.b09-2.el7_8.x86_64/bin/:$JAVA_HOME/bin:$JAVA_HOME/jre/bin:$PATH:$HOME/.local/bin:$HOME/bin:$HOME/pig/bin

# Add PIG_CLASSPATH for pig installation
export PIG_CLASSPATH=/home/vagrant/hadoop/etc/hadoop
export PIG_USER_CLASSPATH_FIRST=/home/vagrant/pig/lib


export HIVE_HOME=/home/vagrant/hive
export PATH=$PATH:$HIVE_HOME/bin
export CLASSPATH=$CLASSPATH:$HIVE_HOME/lib
```

**Comment by gitava** [comment]
none of above is critical ...it's because the following jar file is missing. After put it back, everything is OK now.

```sh
[vagrant@hdp-node-01 lib]$ pwd
/home/vagrant/jdk/lib
[vagrant@hdp-node-01 lib]$ ls -l tools.jar
-rw-rw-r--. 1 vagrant vagrant 15251138 Apr 11  2015 tools.jar
```
